test/test.c

```diff
@@ -13627,12 +13730,14 @@ test_SlowAsyncSubscriber(void)
     CHECK_SERVER_STARTED(serverPid);
 
     s = natsConnection_Connect(&nc, opts);
-    IFOK(s, natsConnection_Subscribe(&sub, nc, "foo", _recvTestString, (void*) &arg));
+    IFOK(s, natsConnection_Subscribe(&sub, nc, "foo.*", _recvTestString, (void*) &arg));
```

Comment:
not a functional change but was helpful in debigging to make the messages distinct.